polkadot-parachains/encointer-runtime/src/weights/pallet_encointer_ceremonies.rs

```diff
@@ -0,0 +1,150 @@
+	// Storage: EncointerScheduler CurrentCeremonyIndex (r:1 w:0)
+	// Storage: EncointerCommunities NominalIncome (r:1 w:0)
+	// Storage: EncointerCeremonies CeremonyReward (r:1 w:0)
+	// Storage: EncointerCeremonies MeetupCount (r:1 w:0)
+	// Storage: EncointerCeremonies AssignmentCounts (r:1 w:0)
+	// Storage: EncointerCeremonies Assignments (r:1 w:0)
+	// Storage: EncointerCeremonies BootstrapperIndex (r:1 w:0)
+	// Storage: EncointerCeremonies ReputableIndex (r:1 w:0)
+	// Storage: EncointerCeremonies IssuedRewards (r:1 w:1)
+	// Storage: EncointerCeremonies ReputableRegistry (r:8 w:0)
+	// Storage: EncointerCeremonies NewbieRegistry (r:2 w:0)
+	// Storage: EncointerCeremonies MeetupParticipantCountVote (r:10 w:0)
+	// Storage: EncointerCeremonies AttestationIndex (r:10 w:0)
+	// Storage: EncointerCeremonies AttestationRegistry (r:10 w:0)
+	// Storage: EncointerBalances Balance (r:10 w:10)
+	// Storage: EncointerBalances DemurragePerBlock (r:1 w:0)
+	// Storage: EncointerBalances TotalIssuance (r:1 w:1)
+	// Storage: EncointerCeremonies ParticipantReputation (r:0 w:10)
+	fn claim_rewards() -> Weight {
+		(1_173_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(62 as Weight))
+			.saturating_add(T::DbWeight::get().writes(22 as Weight))
+	}
+	// Storage: EncointerCeremonies InactivityTimeout (r:0 w:1)
+	fn set_inactivity_timeout() -> Weight {
+		(22_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: EncointerScheduler CurrentPhase (r:1 w:0)
+	// Storage: EncointerCeremonies MeetupTimeOffset (r:0 w:1)
+	fn set_meetup_time_offset() -> Weight {
+		(25_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(1 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: EncointerCeremonies ReputationLifetime (r:0 w:1)
+	fn set_reputation_lifetime() -> Weight {
+		(21_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: EncointerCeremonies EndorsementTicketsPerBootstrapper (r:0 w:1)
+	fn set_endorsement_tickets_per_bootstrapper() -> Weight {
+		(22_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: EncointerCeremonies Assignments (r:0 w:1)
+	// Storage: EncointerCeremonies ParticipantReputation (r:0 w:1)
+	// Storage: EncointerCeremonies MeetupCount (r:0 w:1)
+	// Storage: EncointerCeremonies ReputableRegistry (r:0 w:1)
+	// Storage: EncointerCeremonies NewbieCount (r:0 w:1)
+	// Storage: EncointerCeremonies ReputableCount (r:0 w:1)
+	// Storage: EncointerCeremonies AssignmentCounts (r:0 w:1)
+	// Storage: EncointerCeremonies BootstrapperCount (r:0 w:1)
+	// Storage: EncointerCeremonies ReputableIndex (r:0 w:1)
+	// Storage: EncointerCeremonies AttestationCount (r:0 w:1)
+	// Storage: EncointerCeremonies EndorseeCount (r:0 w:1)
+	// Storage: EncointerCeremonies EndorseesCount (r:0 w:1)
+	fn purge_community_ceremony() -> Weight {
+		(131_000_000 as Weight)
```

Comment:
@brenzi I forgot about this one, do we want to update that too similar to the encointer-node?

https://github.com/encointer/encointer-node/commit/8ac46d1aec5ccace957e959e6a1f69f372f149eb

btw, what was your bad experience?

Reply:
bad experience was that the block time went up to several minutes while batch-purging with 800 purge extrinsics in one block (which means weight calculation was terribly wrong. No big problem for a solochain. But a parachain is bricked if block processing takes too long. So I'd rather manualley set a very high weight before we have better benchmarking

Reply:
just set it equal to the node: 10^10